salt/modules/mod_random.py

```diff
@@ -144,3 +144,29 @@ def rand_int(start=1, end=10):
         salt '*' random.rand_int 1 10
     '''
     return random.randint(start, end)
+
+
+def seed(range=10, hash=None):
+    '''
+    Returns a random number within a range. Optional hash argument can
+    be any hashable object. If hash is omitted or None, the id of the minion is used.
+
+    .. versionadded: 2015.5.3
+
```

Comment:
@aboe76, either this `versionadded` needs to be Beryllium or this pull request needs to be applied to the 2015.5 branch.  Which do you prefer?